packages/create-app/templates/default-app/packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -157,6 +157,13 @@ const websiteEntityPage = (
   </EntityLayout>
 );
 
+/**
+ * TODO: For the MUI Grid to work there have to be "xs" set on every GridItem,
+ * such that the seperation of space is clear from the smallest screen size upwards
+ * https://material-ui.com/components/grid/#basic-grid.
```

Comment:
```suggestion
 * NOTE: This page is designed to work on small screens such as mobile devices.
 * This is based on Material UI Grid; each grid item must have an `xs` column size
 * set, since this does not default.
 * https://material-ui.com/components/grid/#basic-grid.
```

I think this is a note for consumers rather than a TODO, right? I put in a suggested rewording, trying to make it a bit clearer.